README.md

```diff
@@ -31,17 +31,48 @@ __gtk__ expects __GTK+__, __GLib__ and __Cairo__ development files to be install
 
 ### Windows
 
+Make sure you have the [GNU
+ABI](https://www.rust-lang.org/downloads.html#win-foot) version of the rust
+compiler installed.  Since we need our own toolchain make sure to uncheck
+"Linker and platform libraries" in the Rust setup.
+![Screenshot](rust_setup.png)
+
+If you already installed Rust, check if your Rust installation has `gcc.exe`
+and `ld.exe` in its `bin` directory.  In that case remove those executables,
```

Comment:
I assume you mean I should remove the double space? That's actually my vim setup auto formatting the sentences with two spaces after the period.